### PR TITLE
added note creation

### PR DESCRIPTION
**Pull request by kti-sam**

### CODE REVIEW

File: djautotask/models.py [hunk 1]
```diff
@@ -302,35 +305,49 @@ def __str__(self):
         return '{} {}'.format(self.resource, self.ticket)
 
 
-class TicketNote(TimeStampedModel):
+class Note:
+    ALL_USERS = 1
+    INTERNAL_USERS = 2
+    PUBLISH_CHOICES = ((ALL_USERS, 'All Autotask Users'),
+                       (INTERNAL_USERS, 'Internal Project Team'))
```
Comment (kti-cameron) [inline djautotask/models.py:312]: Should `Internal Project Team` be `Internal Users` instead? I haven't seen any mention of `Internal Project Team` in the AT UI.

Comment (kti-sam) [reply djautotask/models.py:312]: Ah oops, good catch. I pulled that from the API actually. Thats what the picklist field reported it as.

Comment (kti-cameron) [reply djautotask/models.py:312]: Oh yeah I see that now. That confusing

Comment (kti-sam) [reply djautotask/models.py:312]: I changed it anyway so it makes more sense to Autotask users, since its "Internal Project Team" even for Tickets.

**Pull request merged**